cfgov/v1/models/ref.py

```diff
@@ -121,6 +130,9 @@ def category_label(category):
             if category == slug:
                 return name
 
+def fcm_label(category):
+    for slug, name in fcm_types:
+            return name
```

Comment:
This always returns the first `name` it sets! I think you're looking to return the name _if_ the `slug` and the given `category` are equal.